less/angular/pages/goggles.less

```diff
@@ -0,0 +1,189 @@
+.goggles-backgrounded {
+  position: relative;
+  z-index: 1;
+  background: inherit;
+  overflow: hidden;
+
+  &.goggles-xray-image {
+    padding-bottom: 3em;
+
+    img {
+      position: absolute;
+      z-index: 0;
+      padding: 0;
+    }
+  }
+
+  .goggles-header {
+    display: flex;
+    flex-direction: row;
+
+    position: relative;
+    background: white;
+    width: 100%;
+    padding: 3em 0 2em;
+
+    h1 {
+      flex: auto;
+      align-self: stretch;
+      margin: 0;
+    }
+
+    .goggles-bookmark {
+      flex: 0 1 auto;
+      align-self: stretch;
+      text-align: right;
+
+      p {
+        text-align: center;
+      }
+    }
+
+    .goggles-bookmark-link {
+      color: black;
+      text-decoration: none;
+      font-weight: bold;
+
+      &:hover, &:active, &:visited {
```

Comment:
Best practice to add `&:link` for backwards compatibility (INTERNET EXPLORER) and `&:focus` for a11y support (in this order… `&:link, &:hover, &:active, &:visited, &:focus`)